situp-plugins/apmtracesource/src/test/java/com/amazon/situp/plugins/source/oteltrace/OTelTraceSourceTest.java

```diff
@@ -141,4 +142,22 @@ void testHttpFullBytes() {
                     validateBuffer();
                 });
     }
+
+    @Test
+    public void testDoubleStart() {
+        Assertions.assertThrows(IllegalStateException.class, () -> SOURCE.start(BUFFER));
```

Comment:
Not sure why this throws IllegalStateException while BUFFER is not null?

Reply:
This is because, we are trying to start a server that is already started. So it throws IllegalStateExeception.

Reply:
Have we and should we explicitly capture this exception in the `start()`? b/c otherwise this seems to be testing armeria instead of our API.